public/js/UnleashApp.jsx

```diff
@@ -1,69 +1,67 @@
-var React                   = require('react');
-var Router                  = require('react-router');
-var Menu                    = require('./components/Menu');
-var ErrorMessages           = require('./components/ErrorMessages');
-var initalizer              = require('./stores/initalizer');
-var FeatureToggleStore      = require('./stores/FeatureToggleStore');
-var StrategyStore           = require('./stores/StrategyStore');
-var ArchiveStore            = require('./stores/ArchivedToggleStore');
-var Link = Router.Link;
-var RouteHandler = Router.RouteHandler;
+import React from 'react'
+import { Link, Router, RouteHandler } from 'react-router'
+import Menu from './components/Menu'
+import ErrorMessages from './components/ErrorMessages'
+import initalizer from './stores/initalizer'
+import FeatureToggleStore from './stores/FeatureToggleStore'
+import StrategyStore from './stores/StrategyStore'
+import ArchiveStore  from './stores/ArchivedToggleStore'
 
-var UnleashApp = React.createClass({
+const UnleashApp = React.createClass({
```

Comment:
Probably, but I left that out, since there's a lot of fragile code at the current build, so I left that out for you to decide :smile: